src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  const callSum = function getSum(a) {
+    count++;
+
+    if ((count > 3) && (count % 2 === 0)) {
+      return () => 'Bzzz... Error!';
+    } else {
```

Comment:
if you're returning in if statement above there is no point in using else, just return